app/actions/remote/entry/gql_common.ts

```diff
@@ -202,7 +216,7 @@ export const entryGQL = async (serverUrl: string, currentTeamId?: string, curren
     let initialTeamId = currentTeamId;
     if (!teamData.teams.length) {
         initialTeamId = '';
-    } else if (!initialTeamId || !teamData.teams.find((t) => t.id === currentTeamId)) {
+    } else if (!initialTeamId || !teamData.teams.find((t) => t.id === currentTeamId && t.delete_at === 0)) {
```

Comment:
0/5 This might be trivial and not used often, but perhaps an `isArchived` function for teams similar to the one for channels `app/utils/channel/index.ts`